ptx/sec_deriv_intro.ptx

```diff
@@ -594,10 +594,10 @@
           <!-- START figures/fig_normal1.tex -->
           <image xml:id="img_normal1">
             <shortdescription>
-              A graph of function <m> 3x^2+5x-7</m> along with its normal line at <m>x=1</m>.
+              A graph of function 3x^2+5x-7 along with its normal line at x=1.
```

Comment:
You could also just say, "The graph of the function for this example, along with its normal line at x=1."